gitconfig.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+gitConfig () {
+    git config --global user.email "$email"
+    git config --global user.name "$username"
+    git config --global push.default simple
+}
+
+read -p "What is your real name? " username
+read -p "What is your Github email address? " email
```

Comment:
I am inclined to say that we put some email regex validation here, but at the same time, if the Git command automatically does some basic validation, we should leave it to the `git` command.